actions/commit-and-push/action.yaml

```
@@ -22,6 +22,10 @@ inputs:
     description: "Whether to add untracked files to commit."
     required: false
     default: "false"
+  pass-empty-commit:
+    description: "Whether to exit with code 0 when nothing to commit."
+    required: false
+    default: "false"
```

A boolean would be better here and GitHub actions supports it. Please shortly test again after changing:
```suggestion
    default: false
```

---

WITH the quotes:

[pass-empty-commit=true, Non-empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594742711/jobs/8114152450)
[pass-empty-commit=true, Empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594745065/jobs/8114158026)
[pass-empty-commit=false, Non-empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594727019/jobs/8114115428)
[pass-empty-commit=false, Empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594731254/jobs/8114125237)


WITHOUT quotes:

[pass-empty-commit=true, Non-empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594801621)
[pass-empty-commit=true, Empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594808915)
[pass-empty-commit=false, Non-empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594823872)
[pass-empty-commit=false, Empty commit](https://github.com/bakdata/github-playground-2/actions/runs/4594831337)